apps/prairielearn/elements/pl-checkbox/pl-checkbox.py

```diff
@@ -10,7 +11,7 @@
 
 
 class PartialCreditType(Enum):
-    ALL_OR_NOTHING = "none"
+    OFF = "none"
     NET_CORRECT = "PC"
```

Comment:
these enum shortcuts need to be renamed

Reply:
@reteps what should these be changed to?

Reply:
I'm thinking `none`, `NET`, `EACH`, `COV`